src/index.cpp

```diff
@@ -4935,7 +4935,7 @@ static void writeIndex(OutputList &ol)
       ol.startTitleHead(QCString());
       ol.generateDoc(Doxygen::mainPage->docFile(),Doxygen::mainPage->getStartBodyLine(),
                   Doxygen::mainPage.get(),nullptr,Doxygen::mainPage->title(),false,false,
-                  QCString(),true,false,Config_getBool(MARKDOWN_SUPPORT),false);
+                  QCString(),true,false,Config_getBool(MARKDOWN_SUPPORT));
```

Comment:
the `false` parameter is needed for title's, so this one should stay, the change below is correct.